libs/domain/site/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from 'lit';
+
+export const styles = css`
+  oryx-button {
+    --_height: 38px;
```

Comment:
It would be great to understand what are other use cases for this component. But I'd add `type` or `variation` with one value right now where we can attach current design.

Reply:
But why we are adding an option to another component, not to menu-item?

Reply:
Attribute is now on the host component.